app/helpers/trainee_helper.rb

```diff
@@ -16,14 +16,20 @@ def view_trainee(trainee)
     end
   end
 
-  def trainees_page_title(trainees)
+  def trainees_page_title(trainees, total_trainees_count)
     total_pages = trainees.total_pages
-    return "trainees.index" if total_pages <= 1
+    total_trainees_count_text = total_trainees_count.to_s + " record".pluralize(total_trainees_count)
+
+    if total_pages <= 1
+      return I18n.t("components.page_titles.trainees.index",
+                    total_trainees_count_text: total_trainees_count_text)
```

Comment:
Just to keep the indentation style consistent with the below:

```suggestion
      return I18n.t(
        "components.page_titles.trainees.index",
        total_trainees_count_text: total_trainees_count_text,
      )
```